hack/tests/e2e-ansible.sh

```diff
@@ -145,12 +145,16 @@ cat "$ROOTDIR/test/ansible-memcached/watches-foo-kind.yaml" >> memcached-operato
 pushd memcached-operator
 # Add a second Kind to test watching multiple GVKs
 operator-sdk add crd --kind=Foo --api-version=ansible.example.com/v1alpha1
-sed -i 's|\(FROM quay.io/operator-framework/ansible-operator\)\(:.*\)\?|\1:dev|g' build/Dockerfile
+
+sed -i".bak" -E -e 's|\(FROM quay.io/operator-framework/ansible-operator\)\(:.*\)\?|\1:dev|g' build/Dockerfile; rm -f build/Dockerfile.bak
+
 operator-sdk build "$DEST_IMAGE"
 # If using a kind cluster, load the image into all nodes.
 load_image_if_kind "$DEST_IMAGE"
-sed -i "s|{{ REPLACE_IMAGE }}|$DEST_IMAGE|g" deploy/operator.yaml
-sed -i 's|{{ pull_policy.default..Always.. }}|Never|g' deploy/operator.yaml
+sed -i".bak" -E -e "s|{{ REPLACE_IMAGE }}|$DEST_IMAGE|g" deploy/operator.yaml; rm -f deploy/operator.yam.bak
+sed -i".bak" -E -e 's|{{ pull_policy.default..Always.. }}|Never|g' deploy/operator.yaml; rm -f deploy/operator.yam.bak
+pushd "${ROOTDIR}/test/ansible-inventory"
```

Comment:
Do we need to change directory into `"${ROOTDIR}/test/ansible-inventory"`? Regardless, this change seems out of scope for this PR.